tests/db_functions/comparison/test_cast.py

```diff
@@ -51,6 +51,34 @@ def test_cast_to_integer(self):
                 numbers = Author.objects.annotate(cast_int=Cast('alias', field_class()))
                 self.assertEqual(numbers.get().cast_int, 1)
 
+    def test_case_from_db_datetime_to_date(self):
+        DTModel.objects.create(start_datetime=datetime.datetime.now())
```

Comment:
I wonder if it wouldn't be better to use a specific datetime so the return value could be verified rather than just the type?